app/src/pages/Flow/Nodes/Dashboard.tsx

```diff
@@ -64,21 +60,28 @@ const PreviewRenderer = ({ label, node }: PreviewProps) => (
   </div>
 );
 
+const nodeInputs: HvFlowNodeInput[] = [
+  {
+    label: "Visualizations",
+    isMandatory: true,
+    accepts: ["visualizations"],
+  },
+];
+
 export const Dashboard: HvFlowNodeFC = (props) => {
   const { id } = props;
 
   const { nodeTypes } = useFlowContext();
 
-  const [configuration, setConfiguration] = useState<Configuration>({
-    opened: false,
-  });
+  const [open, setOpen] = useState(false);
+  const [config, setConfig] = useState<DashboardSpecs>();
   const [content, setContent] = useState<PreviewProps[]>();
 
   const handleOpenConfig = () => {
     // Get from local storage
     const value = localStorage.getItem(DASHBOARDS_STORAGE_KEY);
     const specs: DashboardsStorage = value ? JSON.parse(value) : undefined;
-    const config = specs?.[id];
+    const newConfig = specs?.[id];
 
     const ct = config?.items?.map<PreviewProps>((node) => {
```

Comment:
When clicking on the configure button, the "No visualizations connected to the dashboard." message appears because of:

```suggestion
    const ct = newConfig?.items?.map<PreviewProps>((node) => {
```

Reply:
My bad 🤦 I messed up the renaming